addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -115,7 +115,10 @@ export class DiscussCoreCommon {
                     channel.selfMember.syncUnread = true;
                     channel.scrollUnread = true;
                 }
-                if (notifId > channel.selfMember?.message_unread_counter_bus_id) {
+                if (
+                    notifId > channel.selfMember?.message_unread_counter_bus_id &&
+                    !message.isNotification
+                ) {
```

Comment:
Shouldn't this require a server change too? If the value is fetched from the server won't it include the notif messages?

Reply:
No there was a mismatch, `_compute_message_unread` excludes notifications.